tests/ci/host_test.sh

```diff
@@ -6,5 +6,18 @@ set -ev
 
 cd $TRAVIS_BUILD_DIR/tests/host
 
-make CI
+
+make -j2 ssl
+for i in ../../libraries/ESP8266WiFi/examples/WiFiClient/WiFiClient \
+	../../libraries/ESP8266WiFi/examples/BearSSL_Validation/BearSSL_Validation \
+	../../libraries/ESP8266WebServer/examples/HelloServer/HelloServer \
+	../../libraries/SD/examples/Files/Files \
+	../../libraries/LittleFS/examples/LittleFS_Timestamp/LittleFS_Timestamp \
+	../../libraries/LittleFS/examples/SpeedTest/SpeedTest ; do
+	make -j2 D=1 $i
```

Comment:
In a 64-bits gcc-multilib environment, 32 bits binaries are generated by default but valgrind does not work with them. `FORCE32=0` disables that and 64 bits binaries are generated.
In a non-gcc-multilib 64-bits environment, everything is 64 bits.
Just in case, I would `make -j2 D=1 FORCE32=0 $i`